doc/tutorials/validation.md

```diff
@@ -46,7 +46,7 @@ The most direct way to validate keys is
 sudo kdb mount validation.dump user/tutorial/together dump validation
 kdb vset user/tutorial/together/test 123 "[1-9][0-9]*" "Not a number"
 kdb set user/tutorial/together/test abc
-# STDERR: The command kdb set failed while accessing the key database .*⏎
+# STDERR: The command kdb(.*) set failed while accessing the key database .*⏎
```

Comment:
shouldn't we use "ERROR: 42" here instead?

Reply:
Correct. This problem was already [fixed](https://github.com/ElektraInitiative/libelektra/pull/1722/commits/e9f19ebdb6b66ab9c85994c1a1d8667a92527948#diff-0f1c0a8e1b4271990cf17b6a1472af8aR50) in pull request #1722 though.